cmd/process_test.go

```diff
@@ -24,7 +24,7 @@ func TestProcessNewLog(t *testing.T) {
 	sequenceMap := types.NewSequenceMap()
 	processingQueue = make(chan *types.TxState, 10)
 
-	go cmd.StartProcessor(context.TODO(), a, registeredDomains, processingQueue, sequenceMap)
+	go cmd.StartProcessor(context.TODO(), a, registeredDomains, processingQueue, sequenceMap, nil)
```

Comment:
every instance of `StartProcessor` in this PR now passes nil. Is the arg: sequenceMap still required for noble or ETH?

Reply:
yes, is used in Start(a *AppState) only looks like

Reply:
Yes. It was either passing nil or mocking out a no-op prometheus metrics for testing.